salt/kube-apiserver/apiserver.jinja

```diff
@@ -6,7 +6,7 @@
 
 {% set cloud_provider = salt.caasp_pillar.get('cloud:provider') %}
 {% set kube_admission_control = ['Initializers', 'NamespaceLifecycle', 'LimitRanger', 'ServiceAccount', 'NodeRestriction', 'PersistentVolumeLabel', 'DefaultStorageClass', 'ResourceQuota', 'DefaultTolerationSeconds'] %}
-
+{% set audit_log = salt.caasp_pillar.get('api:audit:log:maxage') and salt.caasp_pillar.get('api:audit:log:maxsize') and salt.caasp_pillar.get('api:audit:log:maxbackup') and salt.caasp_pillar.get('api:audit:log:policy') %}
```

Comment:
Wouldn't it be better to add a `api:audit:enabled` flag? It would make things easier and, at the same time, we could provide some safe defaults for these attributes while the audit feature could be disabled by default.

Reply:
I will change that